release-content/release_notes_template.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Feature name
+authors: ["@FerrisTheCrab", "@BirdObsessed"]
+pull_requests: [14791, 15458, 15269]
+---
+
+Copy the contents of this file into `./release-notes`, update the metadata, and add release note content here. You may use markdown headings levels one through three, and must not start with a heading.
```

Comment:
```suggestion
Copy the contents of this file into `./release-notes`, update the metadata, and add release note content here. You may use markdown headings starting from H3 (`###`), and must not start with a heading. Use backticks when mentioning types either in the title or the content body (e.g. `Vec<T>`), specially for the ones with generics as they'll be interpreted as HTML.
```

H1 will already be used in the page for the post title and H2 for the front-matter `title`.

Reply:
My tool will translate these into headings into the correct level during the merge step.